### PR TITLE
Include missing groupby functions in documentation

### PR DESCRIPTION
**Pull request by quasiben**
## Description
Include missing groupby functions in documentation

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

**Review by shwina — APPROVED** [review]

File: docs/cudf/source/api_docs/groupby.rst [hunk 1]
```diff
@@ -48,15 +48,24 @@ Computations / descriptive stats
    GroupBy.cummax
    GroupBy.cummin
    GroupBy.cumsum
+   GroupBy.diff
    GroupBy.ffill
+   GroupBy.first
+   GroupBy.get_groups
```
Comment (galipremsagar) [inline docs/cudf/source/api_docs/groupby.rst:54]:
```suggestion
```

Grep shows we don't have a `get_groups` yet on the python side. But we have a c++ API for it.

Comment (galipremsagar) [reply docs/cudf/source/api_docs/groupby.rst:54]: never mind you probably want `get_group` here

Comment (galipremsagar) [inline docs/cudf/source/api_docs/groupby.rst:54]:
```suggestion
   GroupBy.get_group
```

**Review by galipremsagar — APPROVED** [review]

**Pull request merged**